### ISSUE TITLE
classification with noisy labels

### ISSUE THREAD

**Issue by dalessioluca** [broadinstitute/ML_for_slideseq#19]
https://github.com/songhwanjun/Awesome-Noisy-Labels
https://github.com/subeeshvasu/Awesome-Learning-with-Label-Noise

**Comment by dalessioluca** [comment]
I will try:
- 2020 - Learning from Noisy Labels with Deep Neural Networks: A Survey. [Paper] [Project Page]
- 2020-Arxiv - Contrast to Divide: self-supervised pre-training for learning with noisy labels. [Paper] [Code]
- A Framework using Contrastive Learning for Classification with Noisy Labels https://arxiv.org/pdf/2104.09563.pdf
